packages/taro-service/src/utils/index.ts

```
@@ -117,3 +117,19 @@ export function printHelpLog (command, optionsList: Map<string, string>, synopsi
   }
 }
+
+export function filterGlobalConfig (globalConfig: IProjectConfig, command: string) {
+  if (!command) {
+    return globalConfig
+  }
+  const config = globalConfig
+
+  const RelatedPluginTag = `@jdtaro/plugin-${command}`
+  if (config.plugins) {
+    config.plugins = config.plugins.filter(pluginName => {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)